libraries/core-react/stories/components/Dialog.stories.tsx

```diff
@@ -48,139 +38,154 @@ const Placeholder = styled.div`
+                  Lorem ipsum dolor sit amet consecteur dit lot. Lorem ipsum
+                  dolor sit amet consecteur dit lot. Lorem ipsum dolor sit amet
+                  consecteur dit lot.
+                </Typography>
+                <Typography variant="body_short">
+                  Lorem ipsum dolor sit amet consecteur dit lot. Lorem ipsum
+                  dolor sit amet consecteur dit lot. Lorem ipsum dolor sit amet
+                  consecteur dit lot.
+                </Typography>
+              </>
             </Dialog.CustomContent>
             <Dialog.Actions>
-              <TempButtonWrapper>
+              <Wrapper>
                 <Button variant="ghost" onClick={() => setVisibleScrim(false)}>
                   Cancel
                 </Button>
                 <Button onClick={() => setVisibleScrim(false)}>OK</Button>
-              </TempButtonWrapper>
+              </Wrapper>
             </Dialog.Actions>
           </Dialog>
         </Scrim>
       )}
-    </Body>
+    </>
   )
 }
 
 export const TextPlusAction: Story<DialogProps> = () => (
   <Dialog>
     <Dialog.Title>Text + actions</Dialog.Title>
     <Dialog.CustomContent>
-      {CUSTOM_CONTENT_CHOICES['description']}
+      <Typography variant="body_short">Small description here.</Typography>{' '}
     </Dialog.CustomContent>
-    <Dialog.Actions>{ACTION_CHOICES['buttons']}</Dialog.Actions>
+    <Dialog.Actions>
+      {' '}
+      <Wrapper>
+        <Button>OK</Button>
+        <Button variant="ghost">Cancel</Button>
+      </Wrapper>
+    </Dialog.Actions>
   </Dialog>
 )
 
 export const PlaceholderPlusAction: Story<DialogProps> = () => (
   <Dialog>
     <Dialog.Title>Placeholder + actions</Dialog.Title>
     <Dialog.CustomContent>
-      {CUSTOM_CONTENT_CHOICES['empty']}
+      <Placeholder>Custom content</Placeholder>
     </Dialog.CustomContent>
-    <Dialog.Actions>{ACTION_CHOICES['buttons']}</Dialog.Actions>
+    <Dialog.Actions>
+      {' '}
+      <Wrapper>
+        <Button>OK</Button>
+        <Button variant="ghost">Cancel</Button>
+      </Wrapper>
+    </Dialog.Actions>
   </Dialog>
 )
 
 export const PlaceholderOnly: Story<DialogProps> = () => (
   <Dialog>
     <Dialog.Title>Placeholder</Dialog.Title>
     <Dialog.CustomContent>
-      {CUSTOM_CONTENT_CHOICES['emptyLarge']}
+      <Placeholder>
+        Custom content in a larger placeholder. No actions, only ESC or
+        timedelay
+      </Placeholder>
     </Dialog.CustomContent>
-    <Dialog.Actions>{ACTION_CHOICES['none']}</Dialog.Actions>
+    <Dialog.Actions>
+      {' '}
+      <Wrapper>
+        <Button>OK</Button>
+        <Button variant="ghost">Cancel</Button>
+      </Wrapper>
+    </Dialog.Actions>
   </Dialog>
 )
 
 export const ScrollablePlusActions: Story<DialogProps> = () => (
   <Dialog>
     <Dialog.Title>Scrollable + actions</Dialog.Title>
     <Dialog.CustomContent scrollable>
-      {CUSTOM_CONTENT_CHOICES['scroll']}
+      <>
```

Comment:
Do we need the fragment tag here?

Reply:
Hopefully not. Copied from the example. Can try to remove it 👍